src/Generator/Generators/CLI/CLIMarshal.cs

```diff
@@ -249,7 +249,7 @@ public override bool VisitTypedefType(TypedefType typedef, TypeQualifiers quals)
             FunctionType function;
             if (decl.Type.IsPointerTo(out function))
             {
-                Context.Return.Write("safe_cast<{0}>(", typedef);
+                Context.Return.Write("{0} == nullptr ? nullptr : safe_cast<{1}>(", Context.ReturnVarName, typedef);
```

Comment:
We are in the process of switching to interpolated strings so please use them here. You could use the excellent add-ins of http://vsrefactoringessentials.com/ and https://marketplace.visualstudio.com/items?itemName=DustinCampbell.CEssentials, the latter has a refactoring to turn string.Format (which won't work in this particular case, though) to an interpolated string.